.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## PR Summary
+
+<!-- summarize your PR between here and the checklist -->
+
+## PR Checklist
+
+Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
+Please mark anything not applicable to this PR `NA`.
+
+- [ ] PR has a meaningful title
+- [ ] Summarized changes
```

Comment:
FWIW I'm somewhat hesitant to add a PR template because we're in the mode of encouraging more contributors so we want a low barrier to entry.  But I think we can add one that isn't too onerous.  Mainly I wanted @tylerl0706 automatically added as a reviewer of each PR.  :-)

Reply:
I think @bergmeister is talking about requiring the Microsoft copyright header to new files.